### PR TITLE
Feat: New Parser for "yum -C --noplugins repolist" command for containers

### PR DESCRIPTION
**Pull request by TZ3070**
Signed-off-by: Xinting Li <xintli@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

New parser for yum repo list in container

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

**Comment by TZ3070** [comment]
The product details are not needed for the rule's front end resolution, the PR is closed.

**Pull request closed without merging**
